packages/node/src/handlers.ts

```diff
@@ -341,6 +341,8 @@ export function trpcMiddleware(options: SentryTrpcMiddlewareOptions = {}) {
         trpcContext.input = normalize(rawInput);
       }
 
+      // TODO: Can we rewrite this to an attribute? Or set this on the scope?
+      // eslint-disable-next-line deprecation/deprecation
       sentryTransaction.setContext('trpc', trpcContext);
```

Comment:
Any thoughts on this? Not sure how important that is, but I guess we need some way to access the scope of the `sentryTransaction` then we could just put it on there...?